src/EditorFeatures/Core/FindUsages/ClassifiedSpanExtension.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text.Shared.Extensions;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Adornments;
+
+namespace Microsoft.CodeAnalysis.Editor.FindUsages
+{
+    internal static class ClassifiedSpanExtension
+    {
+
+        public static ClassifiedTextElement BuildClassifiedTextElementForClassifiedSpans(this IEnumerable<ClassifiedSpan> classifiedSpans, ITextSnapshot snapshot, ITrackingSpan trackingSpan, CancellationToken cancellationToken)
```

Comment:
can you add test on this method? or quick info?

Reply:
Created #28111 to add unit test for close braces and break down this method